lib/tesla/middleware/authorization_oauth.ex

```diff
@@ -12,6 +12,12 @@ defmodule Tesla.Middleware.AuthorizationOAuth do
   end
 
   def headers do
-    [{"Content-type", "application/json"}, {"Accept", "application/json"}]
+    {:ok, vsn} = :application.get_key(:my_app, :vsn)
+
+    [
+      {"Content-type", "application/json"},
+      {"Accept", "application/json"},
+      {"Procore-Sdk-Version", List.to_string(vsn)}
```

Comment:
What does `List.to_string(vsn)` output. Can you add a comment?

Reply:
I played around with this a bit and moved it. See [b2192d1](https://github.com/procore/elixir-sdk/pull/33/commits/b2192d1a6f4cdc9908d60fef17d40926f7a56e48). Comment added there.